tests/VObject/ITip/BrokerProcessReplyTest.php

```diff
@@ -253,6 +253,73 @@ public function testReplyPartyCrasher(): void
         $this->process($itip, $old, $expected);
     }
 
+    public function testReplyExistingExceptionRecurrenceIdInUTC(): void
```

Comment:
Is this the same test case as `testReplyNewExceptionRecurrenceIdInUTC` further down?

Reply:
No, here the exception event already exists in `$old`. So it is a reply to an existing exception event, which wasn't covered/fixed by the original pull request.